<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InsertAdDlg</class>
 <widget class="QDialog" name="InsertAdDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Insert Ad</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Ad Type</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="image1Radio">
        <property name="text">
         <string>Image 120x600</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="image2Radio">
        <property name="text">
         <string>Image 293x95</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="image3Radio">
        <property name="text">
         <string>Image 240x240</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="text1Radio">
        <property name="text">
         <string>Text 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="text2Radio">
        <property name="text">
         <string>Text 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="mobileRadio">
        <property name="text">
         <string>Mobile</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Ad HTML Code</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Please enter the advertisement HTML code that you were issued:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="htmlEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="registerButton">
       <property name="text">
        <string>Register...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>InsertAdDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
